build.gradle

```diff
@@ -264,10 +264,13 @@ task run(type:Exec) {
 }
 
 def defineDevTask(name, body) {
-    task(name, type: Exec) {
+    task(name, type: JRubyExec) {
         description "Run in the development mode with live reloading. Point your browser to http://localhost:4242/"
-        workingDir 'content'
-        commandLine 'awestruct', '--dev', '--output-dir', '../build/_site'
+        script 'awestruct'
+        scriptArgs '--dev',
```

Comment:
The basics of the awestruct dev server work for me with this change, but live reloading doesn't — presumably because I haven't installed whatever Guard deps.